lib/tasks/avalon.rake

```diff
@@ -89,6 +89,23 @@ namespace :avalon do
         end
       end
     end
+
+    desc "Cleanup failed bookmarks"
+    task bookmark_cleanup: :environment do
+      Bookmark.all.each do |b|
+        if MediaObject.where(id: b.document_id).count > 0
+          puts "Bookmark pid: #{b.id} exists"
+        else
+          puts "Bookmark pid: #{b.id} does not exist. Deleting...."
+          begin
+            b.destroy
+          rescue Exception => e
+            puts "Failed to delete"
```

Comment:
Everything else was puts. I checked before deciding on puts. Now that we're getting to something that will be used in production, maybe we could change everything to logger?